Example/CollectionViewController.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  PlanGrid
+//  https://www.plangrid.com
+//  https://medium.com/plangrid-technology
+//
+//  Documentation
+//  https://plangrid.github.io/ReactiveLists
+//
+//  GitHub
+//  https://github.com/plangrid/ReactiveLists
+//
+//  License
+//  Copyright © 2018-present PlanGrid, Inc.
+//  Released under an MIT license: https://opensource.org/licenses/MIT
+//
+
+import ReactiveLists
+import UIKit
+
+@objc
+class CollectionViewController: UIViewController {
+
+    lazy var collectionView: UICollectionView = {
+        UICollectionView(frame: self.view.frame, collectionViewLayout: UICollectionViewFlowLayout())
+    }()
+
+    var collectionViewDataSource: FluxCollectionViewDataSource?
+    var groups: [UserGroup] = [] {
+        didSet {
+            let model = collectionViewModel(
+                forState: groups,
+                onDeleteClosure: { deletedUser in
+                    // Iterate through the user groups and find the deleted user.
+                    for (index, group) in self.groups.enumerated() {
+                        self.groups[index].users = group.users.filter { $0.uuid != deletedUser.uuid }
+                    }
+            }
+            )
+            self.collectionViewDataSource?.collectionViewModel.value = model
+        }
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        self.title = "Collection View Example"
+
+        self.collectionView.frame = self.view.frame
+        self.collectionView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+        self.view.addSubview(self.collectionView)
+
+        self.collectionView.register(UICollectionViewCell.self, forCellWithReuseIdentifier: "CollectionUserCell")
+        self.collectionViewDataSource = FluxCollectionViewDataSource(shouldDeselectUponSelection: true)
+        self.collectionViewDataSource?.collectionView = self.collectionView
+
+        self.navigationItem.leftBarButtonItem = UIBarButtonItem(
+            title: "Flip", style: .plain, target: self, action: #selector(self.swapSections)
+        )
+        self.navigationItem.rightBarButtonItem = UIBarButtonItem(
+            barButtonSystemItem: .add, target: self, action: #selector(self.addUser)
+        )
```

Comment:
I'd also argue for putting everything above here in the storyboards, too.

It helps keep our example code focused on how to use the framework. 

I think it's really nice when all you have to look at is setting up the data source, etc.